src/lib/litegraph/src/subgraph/ExecutableNodeDTO.ts

```diff
@@ -271,14 +271,8 @@ export class ExecutableNodeDTO implements ExecutableLGraphNode {
       // Bypass nodes by finding first input with matching type
       const matchingIndex = this.#getBypassSlotIndex(slot, type)
 
-      // No input types match
-      if (matchingIndex === -1) {
-        console.debug(
```

Comment:
This might be worth keeping as a `.warn`, if we can make it actionable.
But maybe not...

Reply:
Same for some of the other ones.